tests/test_flex_counters.py

```diff
@@ -373,7 +454,244 @@ def test_remove_trap_group(self, dvs):
         for trap_id in trap_ids:
             assert trap_id not in counters_keys
 
+        self.set_flex_counter_group_status(meta_data['key'], meta_data['name_map'], 'disable')
+
+    def test_update_route_pattern(self, dvs):
+        self.setup_dbs(dvs)
+        self.config_db.create_entry('FLOW_COUNTER_ROUTE_PATTERN', '1.1.0.0/16',
+            {
+                'max_match_count': '30'
+            }
+        )
+        self.create_l3_intf("Ethernet0", "")
+        self.create_l3_intf("Ethernet4", "")
+        self.add_ip_address("Ethernet0", "10.0.0.0/31")
+        self.add_ip_address("Ethernet4", "10.0.0.2/31")
+        self.set_admin_status("Ethernet0", "up")
+        self.set_admin_status("Ethernet4", "up")
+        # set ip address and default route
+        dvs.servers[0].runcmd("ip address add 10.0.0.1/31 dev eth0")
+        dvs.servers[0].runcmd("ip route add default via 10.0.0.0")
+        dvs.servers[1].runcmd("ip address add 10.0.0.3/31 dev eth0")
+        dvs.servers[1].runcmd("ip route add default via 10.0.0.2")
+        dvs.servers[0].runcmd("ping -c 1 10.0.0.3")
+
+        dvs.runcmd("vtysh -c \"configure terminal\" -c \"ip route 1.1.1.0/24 10.0.0.1\"")
```

Comment:
Why do we have to change to vtysh? Is the previous approach to write to APP_DB not working?

Reply:
Yes, it does not work after some merge/rebase...I guess there is some new changes which introduce more strict check.